### DISCUSSION TITLE
Using Zeitwerk to autoload code?

### DISCUSSION THREAD

**Discussion by pineapplethief** [Neodelf/lecter#63] [Q&A]
I think we could add [Zeitwerk](https://github.com/fxn/zeitwerk) support to simplify code loading, we won't have to maintain list of all require's manually in `lecter.rb`.

**Comment by Neodelf** [comment]
@pineapplethief Thank you for your proposition!
In general, I keep the way that gem doesn't have to have external dependencies if it can work without theirs.
Cause we have all `require`s in one file we add the dependency to keep it clean. Will it strange, won't it?
